simulator/hosts/base_darwin.yaml

```diff
@@ -0,0 +1,31 @@
+name:
+  base_darwin
+queries:
+  - pattern: select 1$
+    results: [
+      {"1":"1"}
+    ]
+
+  - pattern: select \* from interface_details id join interface_addresses ia
```

Comment:
is there a way to avoid escaping the `*`?

Reply:
Not that I know of... These are regexes, and `*` is interpreted as usual.

Reply:
maybe we should me these patterns `from interface_details id join interface_addresses ia` and omit the columns?